app/validators/alphanumeric_validator.rb

```diff
@@ -1,34 +0,0 @@
-# AlphanumericValidator validates that there's only alphabetic digits in a set of
-# protected values and alphanumerical digits in another set of protected values.
-class AlphanumericValidator < ActiveModel::EachValidator
-  ALPHANUMERICAL_VALUES = [:company_name, :company_unit].freeze
-  ALPHABETICAL_VALUES   = [:country, :state, :city].freeze
-  PROTECTED_VALUES      = (ALPHANUMERICAL_VALUES + ALPHABETICAL_VALUES).freeze
-
-  def validate_each(record, attribute, value)
-    # Base case: irrelevant pillar value, or nil value.
-    return unless relevant?(record.pillar, value)
-
-    # Pillar values that should contain only alphabetical digits.
-    return if contains?(ALPHABETICAL_VALUES, record.pillar) && value.match(/^[a-zA-Z\s]+$/)
-
-    # Pillar values that can contain characters from "space" to "~" in the
-    # ASCII table.
-    return if contains?(ALPHANUMERICAL_VALUES, record.pillar) && value.match(/^[\x20-\x7e]+$/)
```

Comment:
Heh, luckily, all this code is being deleted in this pull request, so that issue is gone once it merges.